packages/server/lib/models/data.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:enum_to_string/enum_to_string.dart';
+
+abstract class Data {
```

Comment:
Maybe we should use dart package jsonSerializable? It makes json serialization much much easier
[https://pub.dev/packages/json_serializable](https://pub.dev/packages/json_serializable)